tests/directives/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -27,6 +27,7 @@ import a from \"a\";
 a();
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 /* @flow */
+
```

Comment:
It's weird that this empty line wasn't added before. But this is great, this use case came from someone that created an issue :)